target/linux/lantiq/image/Makefile

```diff
@@ -134,9 +136,10 @@ endif
 ifeq ($(SUBTARGET),xway_legacy)
 
 define Device/arcadyan_arv4520pw
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := Easybox 800, WAV-281 - ARV4520PW
```

Comment:
I would try to make this a little nicer with brackets instead of the dash, e.g. `Easybox 800, WAV-281 (ARV4520PW)` or similar.

Reply:
Resolved with *_ALT* variables.